app/views/helpers/AmountToPaySummary.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import models.Amounts
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.SummaryList
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import utilities.CurrencyFormatter
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object AmountToPaySummary  {
+  def amountToPaySummary(amounts: Amounts)(implicit messages: Messages): SummaryList = {
+
+    val originalReturnTotal: BigDecimal = amounts.originalReturnTotal
+    val newReturnTotal: BigDecimal = amounts.newReturnTotal
+    val accountBalance: BigDecimal = amounts.accountBalance
+    val adjustedAmount: BigDecimal = amounts.adjustedAmount
+
+    SummaryListViewModel(rows = Seq(
+      SummaryListRowViewModel(
+        key = "correctReturn.originalReturnTotal",
+        value = ValueViewModel(HtmlContent(CurrencyFormatter.formatAmountOfMoneyWithPoundSign(originalReturnTotal).replace("-", "&minus;")))
+          .withCssClass("original-return-total sdil-right-align--desktop")
```

Comment:
Should `original-return-total` be a class? I think it should be an id?
Similar for the other three below